src/memkind_arena.c

```diff
@@ -338,6 +338,15 @@ MEMKIND_EXPORT int memkind_arena_create_map(struct memkind *kind,
         if (err) {
             goto exit;
         }
+
+        // setup kind partition
+        snprintf(cmd, sizeof(cmd), "arena.%u.memkind_partition", arena_index);
+        err = jemk_mallctl(cmd, NULL, NULL, (void *)&kind->partition,
+                           sizeof(unsigned));
+        if (err) {
+            goto exit;
```

Comment:
No log message?

Reply:
Not used for extent hooks either - logging could be done in separate PR